tests/integration/conftest.py

```diff
@@ -59,7 +80,7 @@ def logging_setup():
 
 
 @pytest.fixture(scope="session", autouse=True)
-def ethx_init(diff_initial_state):
+def ethx_init(empty_net_state, diff_initial_state):
```

Comment:
I am not sure about this. Which fixture is going to run first? If they run ordered then I think there will be a problem here because the `diff_initial_state` is going to take the empty state as a reference which is wrong. Sorry if I am missing something.

Reply:
Good point. I will amend that.

Reply:
PR https://github.com/nmstate/nmstate/pull/970 is split this change out.
Please discuss there.